medical_operational/README.md

```diff
@@ -0,0 +1,188 @@
+
+*Write Documentation*
+
+Try to comment your code using PEP guidelines, and don’t repeat yourself.
+TODO: Link the oca standars here.
+
+*Submit Feedback*
+
+The best way to send feedback is to file an issue at https://github.com/oca/vertical-medical/issues.
+
+If you are proposing a feature:
+
+* Explain in detail how it would work.
+* Keep the scope as narrow as possible, to make it easier to implement.
+* Remember that this is a volunteer-driven project, and that contributions
+  are welcome :)
+
+Hacking Odoo Medical
+---
+
+1. Clone this repository:
+
+    ```bash
+    $ git clone git@github.com:oca/vertical-medical.git
+    $ cd medical && checkout 8.0
+    ```
+
+2. Create your own branch locally.
+
+    ```bash
+    $ git checkout -b 8.0-your_new_feature_theme
+    ```
+
+3. Push your first change branch to know you start to work on.
+
+    ```bash
+    $ git push -u origin 8.0-your_new_feature_theme
+    ```
+
+4. Prepare your enviroment to work with this repository and the mandatory ones to have all the enviroment ready.
+
+    ```bash
+    $ git clone https://github.com/odoo/odoo.git
+    $ git clone https://github.com/oca/vertical-medical.git
+    $ cd odoo && checkout 8.0
+    $ cd vertical-medical && checkout 8.0
+    ```
+
+5. Create a postgres user (only for this work to avoid problems not related to this enviroment).
+
+    ```bash
+    $ sudo su postgres
+    $ createuser medicaluser -P
+    #put your password just (1) for example.
+    $ createdb medical -U medicaluser -O medicaluser -T remplate0 -E UTF8
+    ```
+
+6. Run the development enviroment.
+
+    ```bash
+    $ cd path/to/odoo/odoo
+    $ ./openerp-server --addons-path=addons/,../medical -r \
+    medicaluser -w 1 --db-filter=medical \
+    -i medical -d medical
+    ```
+
+7. Do your code, and update it passing -u module -d medical (replacing this paramenter above).
+
+8. Before be sure all is ok, we can delete and create db again with -i
+   paramenter to ensure all install correctly.
+
+    ```bash
+    $ sudo su postgres
+    $ dropbd medical
+    $ createdb medical -U medicaluser -O medicaluser -T remplate0 -E UTF8
+    $ ./openerp-server --addons-path=addons/,../medical -r \
+    medicaluser -w 1 --db-filter=medical \
+    -i medical -d medical
+    ```
+
+9. If all is ok installing, please test your enviroment running your code with ‘test-enabled’.
+
+    ```bash
+    $ ./openerp-server --addons-path=addons/,../medical -r \
+    medicaluser -w 1 --db-filter=medical \
+    -i medical -d medical --test-enable
+    ```
+
+**Note:**
+
+    This will take a time, just do it before commit your change and make push.
+
+10. Add your changes to have them versioned.
+
+    ```bash
+    $ git add .
+    ```
+
+11. Commit your changes.
+
+    ```bash
+    $ git commit -m "[TAG] module: what you did"
+    ```
+
+12. Push your first change branch to know you start to work on.
+
+    ```bash
+    $ git push -u origin 8.0-your_new_feature_theme
+    ```
```

Comment:
Warning, if you want this file to be used as description by Odoo, it must be .rst and not .md